ImportTool/RequestGenerator.cs

```
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace ImportTool
+{
+    public static class RequestGenerator
+    {
+        public static async Task<string> GenerateImportRequest(string connectionString, string prefix, int maxFileNumber)
+        {
+            Parameters parameters = new Parameters();
+
+            // add fixed parts
+            parameters.Add("inputFormat", new FhirString("application/fhir+ndjson"));
+            parameters.Add("mode", new FhirString("InitialLoad"));
+
+            CloudStorageAccount storageAccount;
+            if (CloudStorageAccount.TryParse(connectionString, out storageAccount))
+            {
+                CloudBlobClient cloudBlobClient = storageAccount.CreateCloudBlobClient();
+
+                (int count, int succeedCount) = await AddInputPartsFromBlobs(cloudBlobClient, prefix, maxFileNumber, parameters);
+
+                Console.WriteLine($"Total files: {count} traversed.");
+                Console.WriteLine($"Succeed files: {succeedCount} put as input.");
+            }
+
+            return new FhirJsonSerializer().SerializeToString(parameters);
+        }
+
+        private static async Task<Tuple<int, int>> AddInputPartsFromBlobs(CloudBlobClient cloudBlobClient, string prefix, int maxFileNumber, Parameters parameters)
+        {
+            int count = 0;
+            int succeed_count = 0;
```

we do not need to handle failure, just fail fast.